src/main/java/org/opentripplanner/middleware/otp/response/Leg.java

```diff
@@ -0,0 +1,80 @@
+package org.opentripplanner.middleware.otp.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Plan response, itinerary leg information. Produced using http://www.jsonschema2pojo.org/
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Leg {
+
+    @JsonProperty("startTime")
```

Comment:
Are all of these `@JsonProperty` annotations needed?

Reply:
It doesn't look like it. Have removed.